src/utils/cmd_handler.py

```diff
@@ -55,9 +56,9 @@ def target(stdout, stderr):
 
     def kill_process_by_pid(self, pid):
         try:
-            process = psutil.Process(pid)
-            for proc in process.children(recursive=True):
-                proc.kill()
-            process.kill()
+            if sys.platform == 'win32':
+                subprocess.call(['taskkill', '/F', '/T', '/PID', str(pid)])
+            else:
+                os.killpg(pid, signal.SIGKILL)
```

Comment:
Have you tried it?
I did it with <TestTimeLimit>1</TestTimeLimit>
Please check logs
[strange_timeout.txt](https://github.com/zmaslova/dl-benchmark/files/10103096/strange_timeout.txt)

Reply:
os.system(f'pkill -TERM -P {pid}') looks working solution

Reply:
Thanks, this solution is tested, working